wd_helper.au3

```diff
@@ -1414,12 +1414,17 @@ Func _WD_UpdateDriver($sBrowser, $sInstallDir = Default, $bFlag64 = Default, $bF
 							Else
 								; delete webdriver from disk before unpacking to avoid potential problems
 								FileDelete($sInstallDir & $sDriverEXE)
+								Local $bEXEWasFound = False
 								For $FileItem In $FilesInZip ; Check the files in the archive separately
-									If StringRight($FileItem.Name, 4) = ".exe" Then ; extract only EXE files
+									; https://docs.microsoft.com/pl-pl/windows/win32/shell/folder
+									; https://docs.microsoft.com/pl-pl/windows/win32/shell/folder-items
+									; https://docs.microsoft.com/pl-pl/windows/win32/shell/folderitem
```

Comment:
If you intend to leave these in place, then please switch them over to the English equivalents 😃